src/resources/breadcrumb-resources.ts

```diff
@@ -0,0 +1,11 @@
+import { BreadcrumbIF } from '@/interfaces'
+
+export const dashboardBreadcrumb: BreadcrumbIF = {
+  text: 'BC Registries Dashboard',
+  href: `${sessionStorage.getItem('BUSINESSES_URL')}business`
+}
+
+export const staffDashboardBreadcrumb: BreadcrumbIF = {
+  text: 'Staff Dashboard',
+  href: `${sessionStorage.getItem('BUSINESSES_URL')}staff`
```

Comment:
This URL doesn't work for me. It's this, right?
https://dev.bcregistry.ca/business/staff

Reply:
Oops, no, it's this (which works): https://dev.bcregistry.ca/business/auth/staff